packages/plugin-commands-env/src/env.ts

```diff
@@ -98,6 +102,46 @@ export async function handler (opts: NvmNodeCommandOptions, params: string[]) {
     return `Node.js ${nodeVersion as string} is activated
   ${dest} -> ${src}`
   }
+  case 'uninstall': {
+    if (!opts.global) {
+      throw new PnpmError('NOT_IMPLEMENTED_YET', '"pnpm env use <version>" can only be used with the "--global" option currently')
+    }
+
+    const fetch = createFetchFromRegistry(opts)
+    const { releaseChannel, versionSpecifier } = parseNodeEditionSpecifier(params[1])
+    const nodeMirrorBaseUrl = getNodeMirror(opts.rawConfig, releaseChannel)
+    const nodeVersion = await resolveNodeVersion(fetch, versionSpecifier, nodeMirrorBaseUrl)
+    const nodeDir = getNodeVersionsBaseDir(opts.pnpmHomeDir)
+
+    if (!nodeVersion) {
+      throw new PnpmError('COULD_NOT_RESOLVE_NODEJS', `Couldn't find Node.js version matching ${params[1]}`)
+    }
+
+    const versionDir = path.resolve(nodeDir, nodeVersion)
+
+    if (!existsSync(versionDir)) {
+      throw new PnpmError('ENV_NO_NODE_DIRECTORY', `Couldn't find Node.js directory in ${versionDir}`)
+    }
+
+    const nodePath = path.resolve(opts.pnpmHomeDir, 'node')
+    const nodeLink = await fs.readlink(nodePath).catch(() => '')
+
+    if (nodeLink.includes(versionDir)) {
+      const npmPath = path.resolve(opts.pnpmHomeDir, 'npm')
+      const npxPath = path.resolve(opts.pnpmHomeDir, 'npx')
+
+      await Promise.all([
+        await fs.unlink(nodePath),
+        await fs.unlink(npmPath),
+        await fs.unlink(npxPath),
+      ]).catch(() => {})
```

Comment:
I am not sure these errors should be silenced.

Reply:
My initial thought was that if users manually delete `npm/npx/node` symlink, and we don't silence the error - it would break the process completely, and they would have to create those files (or restore) first.
On second thought, I guess it might be unlikely 99% of the time.

Alternatively, instead of throwing these errors, we could inform the user of the error in a form of a console log (`@pnpm/logger` I guess is the best option), this way we get to remove the Node.js version, and users will be notified, that there was an error trying to unlink symlinks. I'm not sure if it's something that you'd prefer, so please let me know what you think. Thanks!

Reply:
You should ignore ENOENT errors
any other errors should not be ignored I believe.